src/view.js

```diff
@@ -238,6 +274,7 @@ function renderBoard(ctrl) {
     tag: 'div',
     attrs: {
       class: 'cg-board',
+      style: 'cursor: ' + ((ctrl.cursor) ? ctrl.cursor : 'pointer'),
```

Comment:
where is this `cursor` coming from? Can this not be done outside of chessground?